parley/README.md

```diff
@@ -52,4 +52,8 @@ Licensed under either of
 
 at your option.
 
+Some files used for tests are under different licenses:
+
+- The font file `Roboto-Regular.ttf` in `/tests/parley/assets/roboto_fonts/` is licensed solely as documented in that folder (and is licensed under the Apache License, Version 2.0).
+
```

Comment:
We also need this section in the top-level README (i.e. the one at `/README.md` - https://github.com/linebender/parley/blob/main/README.md).

Reply:
Fixed